src/libpmem2/pmem2_windows.c

```diff
@@ -35,71 +35,109 @@
  */
 
 #include "libpmem2.h"
+#include "out.h"
 #include "pmem2.h"
 #include "pmem2_utils.h"
 
+#define PMEM2_HIDWORD(x) ((DWORD)((x) >> 32))
+#define PMEM2_LODWORD(x) ((DWORD)((x) & 0xFFFFFFFF))
```

Comment:
Since these definitions are internal to this compilation unit it may be shorter e.g.: HIDWORD.